zebra-consensus/src/transaction/check.rs

```diff
@@ -462,3 +469,30 @@ fn validate_expiry_height_mined(
 
     Ok(())
 }
+
+/// Accepts a transaction, UTXOs in the same block, and spent UTXOs from the chain.
+///
+/// Returns the lowest/earlier height at which every transparent coinbase spend
+/// in the provided [`Transaction`] will be mature,
+///
+/// Returns None if the transaction has no transparent coinbase spends.
+pub fn tx_transparent_coinbase_spends_maturity(
```

Comment:
Please update this function's documentation to match the arguments and return values.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/6510/commits/e6c6c0f33593095568ffef531379d445f2baa290.